Sources/TapestryCore/Utils/DocsUpdater.swift

```diff
@@ -0,0 +1,110 @@
+import Basic
+import SPMUtility
+import class TuistCore.FileHandler
+
+/// Used by `.docsUpdate` `ReleaseAction`
+public protocol DocsUpdating {
+    /// Update `README.md` and `Library.podspec`
+    /// - Parameters:
+    ///     - path: Root path of the library
+    ///     - version: Version to update the docs to
+    func updateDocs(path: AbsolutePath, version: Version) throws
+}
+
+public final class DocsUpdater: DocsUpdating {
```

Comment:
☝️ I think that we're missing some docs to the class and public methods

Reply:
The docs are there, but for the protocol - but added the documentation to the class, too and also its private methods.